packages/shared/lib/services/nango-config.service.ts

```diff
@@ -9,17 +9,18 @@ import type {
     NangoIntegrationDataV2,
     LayoutMode
 } from '../models/NangoConfig.js';
-import type { HTTP_VERB, ServiceResponse } from '../models/Generic.js';
+import type { ServiceResponse } from '../models/Generic.js';
 import { SyncType } from '../models/Sync.js';
 import localFileService from './file/local.service.js';
 import { NangoError } from '../utils/error.js';
-import { determineVersion, getInterval, isJsOrTsType } from '@nangohq/nango-yaml';
-import type { NangoSyncEndpoint, ScriptTypeLiteral } from '@nangohq/types';
-
-export const nangoConfigFile = 'nango.yaml';
-export const SYNC_FILE_EXTENSION = 'js';
-
-export function loadStandardConfig(configData: NangoConfig, showMessages = false, isPublic?: boolean | null): ServiceResponse<StandardNangoConfig[] | null> {
+import { determineVersion, getInterval, isJsOrTsType, parseEndpoint } from '@nangohq/nango-yaml';
+import type { NangoSyncEndpointV2 } from '@nangohq/types';
+
+/**
+ * Legacy parser only used for flows.yaml
+ * TODO: kill this in favor of nango-yaml + transformation to StandardNangoConfig
+ */
+export function loadStandardConfig(configData: NangoConfig): ServiceResponse<StandardNangoConfig[] | null> {
```

Comment:
After all the massive effort regarding nango.yaml parser and code splitting, I realized this file is close to being useless (which is a good thing), so I could remove everything that was used in the CLI before.